src/app/Ranking.js

```diff
@@ -0,0 +1,57 @@
+export class Ranking {
+    constructor(mode) {
+        const availableModes = ['people', 'starships', 'vehicles'];
+        if (!availableModes.includes(mode)) throw `"${mode}" mode is not valid!`;
```

Comment:
O ile ten zapis mi się podoba, to już nie jeden programista przejechał się na nie stosowaniu {} po ifie. Ale trzeba raczej nauczyć się tego na błędach, wiec możesz zostawić i czekac kiedy Cię dopadnie w jakimś przypadku :)